prow/prowjobs/google/exposure-notifications-verification-server/en-verification-server.yaml

```diff
@@ -15,10 +16,21 @@ presubmits:
         env:
         - name: GO111MODULE
           value: "on"
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/gcs-access-service-account/service-account.json
+        - name: GOOGLE_CLOUD_BUCKET
+          value: apollo-server-prow-test-bucket
+        volumeMounts:
+        - name: gcs-access-service-account
+          mountPath: /etc/gcs-access-service-account
         securityContext:
           # We run docker-in-docker which requires privileged mode
           privileged: true
         resources:
           requests:
             cpu: 7
             memory: '16Gi'
+      volumes:
+      - name: gcs-access-service-account
+        secret:
+          secretName: gcs-access-service-account
```

Comment:
nit: Consider using GKE workload identity to avoid using a service account key here.